pkg/chain/ethereum/ethereum.go

```diff
@@ -29,8 +29,23 @@ func (ec *EthereumChain) Address() common.Address {
 // RegisterAsMemberCandidate registers client as a candidate to be selected
 // to a keep.
 func (ec *EthereumChain) RegisterAsMemberCandidate(application common.Address) error {
+	gasEstimate, err := ec.bondedECDSAKeepFactoryContract.RegisterMemberCandidateGasEstimate(application)
+	if err != nil {
+		return fmt.Errorf("failed to estimate gas [%v]", err)
+	}
+
+	// If we have multiple sortition pool join transactions queued - and that
+	// happens when multiple operators gets eligible to join, e.g. after
```

Comment:
"..operators gets eligible.." -> maybe just "operators are eligible.."

Reply:
Maybe *become eligible to join at the same time*? I intend to express that they become eligible at the same time, that's why transactions land in the mempool more or less at the same time.

Reply:
sure, this sounds better